addons/website_hr_recruitment/models/hr_recruitment.py

```diff
@@ -49,9 +50,9 @@ class Job(models.Model):
     _name = 'hr.job'
     _inherit = ['hr.job', 'website.seo.metadata', 'website.published.multi.mixin']
 
+    @mute_logger('odoo.addons.base.models.ir_qweb')
```

Comment:
shouldn't mute_logger be used only for tests ? why is it needed ?

Reply:
I added a warning when there is no template. Here during the installation of the modules, the template is not yet there and there is a warning. To keep the same operation, you do not need a warning.